### PR TITLE
Cmd

### PR DESCRIPTION
**Pull request by YoniKiriaty**

### CODE REVIEW

File: src/main/java/frc/utils/applicationsutils/CMDHandler.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+public class CMDHandler {
+
+    public static final String REPOSITORY_PATH = Path.of("").toAbsolutePath().toString();
+    public static final String PATH_TO_PYTHON_DIRECTORY = REPOSITORY_PATH + "/src/main/python";
+    public static final String PATH_TO_JAVA_DIRECTORY = REPOSITORY_PATH + "/src/main/java";
+
+    public static void runCMDCommand(String command) {
+        Runtime runtime = Runtime.getRuntime();
+        try {
+            runtime.exec(new String[]{"cmd.exe", "/c", command});
```
Comment (hillelv) [inline src/main/java/frc/utils/applicationsutils/CMDHandler.java:15]: magic Strings

File: src/main/java/frc/utils/applicationsutils/OnDeploy.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+public class OnDeploy {
+
+
+    public static void main(String[] args) {
+        startupLogic();
+    }
+
+    private static void startupLogic() {
+        //to be continued...
```
Comment (hillelv) [inline src/main/java/frc/utils/applicationsutils/OnDeploy.java:13]: finish before merging

Comment (YoniKiriaty) [reply src/main/java/frc/utils/applicationsutils/OnDeploy.java:13]: this is for future branches. there is nothing for me to add in this branch.

File: src/main/java/frc/utils/applicationsutils/OnDeploy.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    }
+
+    private static void testConnection() {
+        JFrame frame = new JFrame();
+        frame.setVisible(true);
+        frame.setDefaultCloseOperation(WindowConstants.EXIT_ON_CLOSE);
+    }
```
Comment (YHerm) [inline src/main/java/frc/utils/applicationsutils/OnDeploy.java:20]: What's this? you don't use it

File: src/main/java/frc/utils/applicationsutils/OnDeploy.java [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+import javax.swing.*;
+
+public class OnDeploy {
+
+
+    public static void main(String[] args) {
+        startupLogic();
+    }
+
+    private static void startupLogic() {
```
Comment (YHerm) [inline src/main/java/frc/utils/applicationsutils/OnDeploy.java:12]:
```suggestion
    private static void startCompuerPrograms() {
```
or something in that direction

File: src/main/java/frc/utils/applicationsutils/CMDHandler.java [hunk 5]
```diff
@@ -0,0 +1,24 @@
+package frc.utils.applicationsutils;
+
+import java.io.IOException;
+import java.nio.file.Path;
+
+public class CMDHandler {
```
Comment (YHerm) [inline src/main/java/frc/utils/applicationsutils/CMDHandler.java:6]: consider adding runPythonFile() and runJavaFile() that gets only file name or path, or maybe give the class as param and use getPackageName() (only for java)

File: src/main/java/frc/utils/applicationsutils/OnDeploy.java (hunk 3 above)
Comment (YoniKiriaty) [reply src/main/java/frc/utils/applicationsutils/OnDeploy.java:20]: it's to test if the deploy program is working, it is purely for testing the OnDeploy class.

**Review by hillelv — COMMENTED** [review]
very nice

**Pull request merged**